db/src/main/resources/db/migration/V1_043__create_process_after_timestamp_column.sql

```diff
@@ -0,0 +1,4 @@
+alter table message_queue add column process_after_timestamp timestamp;
+
+-- noinspection SqlWithoutWhere
```

Comment:
Intellij will warn about update statements that don't have a where clause because it will update every row. However, that is exactly what we want in this situation so suppressing this warning.

Reply:
I'd suggest including a not null check on process_after instead

Reply:
Done.